loadtest/README.md

```diff
@@ -0,0 +1,57 @@
+Load Test for Khan API
+======================
+
+This application performs a specified amount of sequential requests to a remote Khan API server, with a specified time period between two consecutive requests. Usage: `../khan loadtest --help`
+
+# Request parameters
+Both amount and period are defined under the key `loadtest` within `../config/local.yaml`:
+```
+loadtest:
+  requests:
+    amount: 1
+    period:
+      ms: 1
+```
+Or setting the following environment variables:
+```
+KHAN_LOADTEST_REQUESTS_AMOUNT (default: 1)
+KHAN_LOADTEST_REQUESTS_PERIOD_MS (default: 1)
+```
+
+# Request operations
+For each request, an operation is chosen at random with probabilities defined at `../config/local.yaml`, like this:
```

Comment:
if the probability is not specified we don't call the operation?

Reply:
Yes! Default value is zero. Default value could be a very small probability instead, but 1 is not an option (will greatly decrease the probabilities of the other operations after sample space normalization).

Reply:
I think we should have sensible default values based on our metrics. If we default to 0 we'll need to think about these probabilities every time someone new runs a load test.

Reply:
Good catch! I'll hardcode default values based on production metrics. Thanks!